src/acl/RegexData.h

```diff
@@ -10,11 +10,10 @@
 #define SQUID_ACLREGEXDATA_H
 
 #include "acl/Data.h"
+#include "base/RegexPattern.h"
 
 #include <list>
 
-class RegexPattern;
-
```

Comment:
This change exposes all src/acl/RegexData.h users to a RegexPattern declaration. Some of those (current or future) users may not need to know how RegexPattern is declared to compile, so we should not create such an unnecessary exposure. Please undo and `#include` RegexPattern.h where it is necessary instead.

Reply:
Done in [485057f](https://github.com/squid-cache/squid/pull/991/commits/485057f53116f54fbc76a6a27f4d41ff11858e29)